File: PTJ3-Text-and-Collection-Port-Lab/starter/cpp/main.cpp
#include <iostream>
#include <string>
#include <vector>

std::vector<std::string> longWords(const std::vector<std::string>& words) {
	// TODO: return only words with length >= 5
	return {};
}

std::string longestWord(const std::vector<std::string>& words) {
	// TODO: return the longest word in the list
	return "";
}

int main() {
	std::vector<std::string> words = {"bridge", "code", "typed", "syntax", "port"};
	auto result = longWords(words);
	for (const auto& word : result)
		std::cout << word << ' ';
	std::cout << "\n" << longestWord(words) << "\n";
}

File: PTJ3-Text-and-Collection-Port-Lab/starter/java/Main.java
import java.util.ArrayList;
import java.util.List;

public class Main {
	static List<String> longWords(List<String> words) {
		// TODO: return a list containing only words with length >= 5
		return new ArrayList<>();
	}

	static String longestWord(List<String> words) {
		// TODO: return the longest word in the list
		return "";
	}

	public static void main(String[] args) {
		List<String> words = List.of("bridge", "code", "typed", "syntax", "port");
		System.out.println(longWords(words));
		System.out.println(longestWord(words));
	}
}
